docs/internals/howto-release-django.txt

```diff
@@ -368,8 +368,11 @@ You're almost done! All that's left to do now is:
    ``VERSION = (1, 5, 2, 'alpha', 0)``.
 
 #. Add the release in `Trac's versions list`_ if necessary (and make it the
-   default if it's a final release). Not all versions are declared;
-   take example on previous releases.
+   default by changing the ``default_version`` setting in the
```

Comment:
"Not all versions are declared; take example on previous releases." could be replaced with something more explicit ... the new "X.Y" version should be added after the alpha release and the default version should be updated after "dot zero."

Reply:
Thanks, updated.